hostdb/hostdb.go

```diff
@@ -99,12 +99,14 @@ type HostAddress struct {
 
 // A Host pairs a host's public key with a set of interactions.
 type Host struct {
-	KnownSince   time.Time           `json:"knownSince"`
-	PublicKey    types.PublicKey     `json:"public_key"`
-	NetAddress   string              `json:"netAddress"`
-	PriceTable   *HostPriceTable     `json:"priceTable"`
-	Settings     *rhpv2.HostSettings `json:"settings"`
-	Interactions Interactions        `json:"interactions"`
+	KnownSince       time.Time          `json:"knownSince"`
+	LastAnnouncement time.Time          `json:"lastAnnouncement"`
+	PublicKey        types.PublicKey    `json:"public_key"`
+	NetAddress       string             `json:"netAddress"`
+	PriceTable       HostPriceTable     `json:"priceTable"`
+	Settings         rhpv2.HostSettings `json:"settings"`
+	Interactions     Interactions       `json:"interactions"`
+	Scanned          bool               `json:"scanned"`
```

Comment:
It could but I think it's not worth saving the extra column for, the check would comprise of isEmpty checks for settings and price tables and I felt having the column was more valuable.. in fact let me add an index instead? Wasn't going to add one at first but if this is my main argument I guess it needs an index 😁

Reply:
I don't mind removing it but there's a lot of value in being able to (easily) query what % of hosts are scanned. If not for us it might prove useful for our users that might be manually querying the database.